.circleci/config.yml

```diff
@@ -157,22 +136,18 @@ jobs:
 
   # Run tests
   nix-test:
-    machine: true
+    docker:
+      - image: nixos/nix:2.3.10
```

Comment:
That doesn't work because postgresql needs to run as non-root - easiest way to get there was to use a plain linux machine

Reply:
Ah, I remember you already told me about that. I remembered *something*, but wasn't sure what exactly. I had hoped that by moving all the "docker" stuff (which might be a problem, too, with docker-in-docker) out to `nix-build`, I'd be lucky... :D

Reply:
We probably should add a comment :-) the nix docker stuff runs completly without docker, cooking the image from scratch. So no risk there!

Reply:
> We probably should add a comment :-) 

Already did so.

> the nix docker stuff runs completly without docker, cooking the image from scratch. So no risk there!

ah.. ha! Does that mean I can switch to the docker executor for `nix-build`?

Reply:
Yes, that should be fine!

Reply:
More info on the from scratch building here: https://github.com/PostgREST/postgrest/blob/main/nix/tools/docker/README.md

That will yield our 3-4mb docker image next release!